backend/apps/accounts/views.py

```
@@ -1098,3 +1101,39 @@ def get(self, request):
             recommended = max(scored_modules, key=lambda m: m["score"])
 
         return Response({"modules": scored_modules, "next_step": recommended})
+
+
+class ChangePasswordView(APIView):
+    permission_classes = [IsAuthenticated]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)